### ISSUE TITLE
Game Crashing after updating Sodium

### ISSUE THREAD

**Issue by devendrapoonia** [MeteorDevelopment/meteor-client#3917]
### Describe the bug

Hello,
So today I updated Sodium mod and ImmediatelyFast. After upating them, game launched and crashed immediately and crash report showed something related to Sodium and Meteor client. So I tried launching without Meteor client and it worked vice versa with Sodium. But when I tried launching game with both mods installed, it crashed immediately. (Crash report attached)

### Steps to reproduce

```bash
1. Install/Update to latest version of sodium.
2. Install latest dev build of Meteor client.
3. Launch game
```

### Link to crash report/log if applicable (https://mclo.gs)

https://mclo.gs/uxW8wDn

### Meteor Version

0.5.4-1871

### Minecraft Version

1.20.1

### Operating System

Windows

### Before submitting a bug report

- [X] This bug wasn't already reported (I have searched bug reports on GitHub).

- [X] This is a valid bug (I am able to reproduce this on the latest dev build).

**Comment by kickTROLL** [comment]
yea same issue here

**Comment by Big-Iron-Cheems** [comment]
Known issue, Sodium 0.5.0 had massive changes which we need to adapt to. 
In the meantime use the precedent version of it.

**Comment by Wide-Cat** [comment]
resolved as of e2a3dc4c6847709691eac765d1fb63b7cb5639e4
